app/src/main/res/layout/item_poll.xml

```diff
@@ -11,8 +11,8 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="6dp"
         android:background="@drawable/poll_option_background"
+        android:maxLines="3"
```

Comment:
There is also a lenght-limit on the poll option text (so we could get rid of this option). But maybe only for Mastodon?

Reply:
I wouldn't overthink this